starter/package-lock.json

```diff
@@ -1513,9 +1781,8 @@
       }
     },
     "@nationalbankbelgium/stark-core": {
-      "version": "10.0.0-beta.3",
-      "resolved": "https://registry.npmjs.org/@nationalbankbelgium/stark-core/-/stark-core-10.0.0-beta.3.tgz",
-      "integrity": "sha512-psDdiA1sesEFsINuSH693A/TuSctYtmsZpdndxcMVsUkR6dVAQMtGXwQik+D8+vHoQXtMGPDdDSr5ntoUWhvnA==",
+      "version": "file:../dist/packages-dist/stark-core/nationalbankbelgium-stark-core-10.0.0-beta.4-f331991e.tgz",
```

Comment:
Can you revert these 2 lines? They should point to the latest version of Stark on NPM ;)